english/bite-the-marshmallow/padded-room/first-blackout.md

```diff
@@ -0,0 +1,8 @@
+The twinkle bursts into flashes of memory that assemble your senses into a perception
+of where you are. Tendrils of deja-vu tug at your mind reel your conscienceness to the disturbing truth. This looks like the padded cell from a movie or television show.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.